test/integration/component/test_project_limits.py

```diff
@@ -845,7 +835,6 @@ def test_06_volumes_per_project(self):
                          account=self.account.name,
                          domainid=self.account.domainid
                          )
-        self.cleanup.append(self.project_1)
```

Comment:
It is cleaned @DaanHoogland but at a later stage, as first the networks need to be cleaned up. As the tests failed with such  errors - `Can't delete the project yet because it has 1 Network to clean up`

Reply:
it shouldn't be added to the clean list with other statements inbtween. This means exceptions can occur before it is added. It must be added immediately.

Reply:
Thanks @DaanHoogland , I'll update the tests with @weizhouapache 's suggestion of using super.